distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -261,7 +265,7 @@ grant codeBase "file:/admin-core-insecuredefaults/org.apache.felix.scr/org.apach
     permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}-", "read";
     permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}-", "read";
     permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}issuer${/}-", "read";
-    permission java.io.FilePermission "${ddf.home.perm}etc${/}users.properties", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}users.properties", "read,write";
```

Comment:
❓ Logging needs permission to _write_ the file?

Reply:
The default user deletion scheduler deletes the users from the file so it needs write access. That's under `admin-core-insecuredefaults` not sure if that's your question

Reply:
Aw, nuts. Ignore me. I missed the collapsed code block in the diff so when I first looked at this, it looked like it was under `grant codeBase "file:/pax-logging-log4j2" {`. 

Obviously, this change is completely correct.

Reply:
❓ Do all other bundles need that write permission though?

Reply:
No, they don't. But this is the balance we're trying to strike of minimizing the number of separate grant blocks we have because it affects performance. So if we can use some engineering judgement and decide it's okay to grant the permission to a couple of bundles that don't need it, that saves us making a brand new grant block.

Reply:
ouch! How are we ever going to simplify this file if we keep adding permissions to code that doesn't need it. We shall never remember why a permission was added. This is going to be un-maintainable.

Reply:
For something like writing to the users.properties file, which can grant any access to the entire machine, I'm leaning towards us creating a new block just for things that need write to that file. In general, yes we should try to keep the number of blocks minimized, but we need to balance that with how dangerous a permission might be. Allowing someone to create a classloader, or something like that, wouldn't be nearly as dangerous as this. Plus we already have a block above for the jaas command, just add the insecure defaults bundle to that one and remove write from this block. Bundles can be in 1 - n blocks.